requirements.txt

```diff
@@ -1,6 +1,6 @@
 colorama==0.3.6
 docopt==0.6.2
-Jinja2
+Jinja2==2.8
```

Comment:
For ensuring at least Jinja2 2.8 is installed, it would be better to add `Jinja2>=2.8`.